cypress/commons/actions/generic/ScenarioParameters.js

```diff
@@ -69,6 +72,18 @@ function discard() {
   getParametersDiscardButton().click();
   getParametersConfirmDiscardButton().click();
 }
+
+function launch(dontAskAgain = false) {
```

Comment:
Is not there a way to factorize launch and launchAndUpdate functions who are barely identical ?

Reply:
Done :+1: